airbyte-cdk/python/airbyte_cdk/sources/declarative/retrievers/simple_retriever.py

```diff
@@ -424,6 +424,7 @@ class SimpleRetrieverTestReadDecorator(SimpleRetriever):
     """
 
     maximum_number_of_slices: int = 5
+    emit_connector_builder_messages: bool = False
```

Comment:
I naively expect this to be true for `SimpleRetrieverTestReadDecorator`. When should we not emit the connector builder messages from the `SimpleRetrieverTestReadDecorator`?

Follow up question: If this should always be true, can we get rid of the field entirely and always emit them in https://github.com/airbytehq/airbyte/pull/24881/files#diff-574e7f982441af95b199fa0d628677b9a654543eaea1534b7516c8679375a235R441?

Reply:
fair point. i think this feeds into the question of the really granular control over the decorator. My initial thought was maybe we want page limits, but not emitting request/response messages. But discussing now I don't really see a use case for not including these because the connector builder grouper needs them. I'll drop this and we'll always have the decorator emit the messages